iac-extensions/arm/src/test/resources/checks/ShortBackupRetentionCheck/Microsoft.Web_sites.bicep

```diff
@@ -0,0 +1,129 @@
+resource noncompliant1 'Microsoft.Web/sites/config@2022-03-01' = {
+  name: 'backup'
+  properties: {
+    __sonar__: 'Sensitive: retention period is less than 30'
```

Comment:
I have never seen this field before, but apparently, we have it in a couple of tests. What's the difference compared to putting description into `name`?

Reply:
In bicep, a resource type can have restrictions on which names are allowed, and depending on the name used, It can define how the resource object behaves.

This is the case with `Microsoft.Web/sites/config`, where one of the names expected is `backup`.
In the original version of how this check S6364 was written, we ensured that the name of the resource was backup, that's also why we have a few tests about it.
This check was later dropped: the other check we performs in the rule are sufficient, there was no upside to ensuring the name was `backup`.

What you see here is mostly an artifact, although real bicep code is indeed expected this resource to have the name `backup`.